bamboo-specs/bamboo.yaml

```diff
@@ -388,6 +389,7 @@ variables:
   EARTHDATA_CLIENT_ID: REPLACE ME
   EARTHDATA_CLIENT_PASSWORD: BAMSCRT@0@0@eYcVOgzAUh3Ar0/flmZ1jQ==  #encrypted
   DB_HOST_ENDPOINT: REPLACE ME
+  BRANCH_NAME: ${bamboo.planRepository.branchName}
```

Comment:
Allows us to pull bucket TF from the current branch. See deployment-rds.sh and deployment-dr-buckets.sh for usages.